src/index.tsx

```diff
@@ -99,6 +100,7 @@ export * from './components/input_fields/textarea_field';
 export * from './components/icons/icon_wrapper';
 export { default as ApprovedIcon } from './components/icons/approved_icon';
 export { default as BuildPlateIcon } from './components/icons/build_plate_icon';
+export { default as BoxIcon } from './components/icons/box_icon';
```

Comment:
Not important for this PR, but if we would use a named export in the first place, we wouldn't have to export all the default exports as a named export here. Thoughts?

Reply:
We've got a rule in eslint that specifically asks for using default exports.

Reply:
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md